src/styles/themes/light.pcss

```diff
@@ -1,69 +1,66 @@
 .light-mode {
-  --main-bg: #fff;
-  --back-bg: #fff;
-  --second-bg: #222B41;
-  --text: #9290a5;
-  --placeholder-text: #28334E;
-  --header: #fff;
-  --header-text: #050505;
-  --button-normal: #2E4C80;
-  --white: #fff;
-  --bright: #050505;
-  --accent: #2E4C80;
-  --green: #54B489;
-  --red: #BD3039;
-  --selected-item: #F0F1FF;
-  --gamboge: #E0A300;
+  /* Primary UI colors */
+  --primary-cta-color: #3C78E0;
+  --contrast-cta-color: #FFF;
 
-  /* linear-gradients */
-  --background-dropdown: var(--selected-item);
-  --background-dropdown-hover: #f5f5ff;
-  --button-dark-gray: rgba(#fff, 0.5);
-  --copyable-field-border: rgba(#28334E, 0.3);
-  --header-border: rgba(#28334E, 0.1);
-  --input-block-border-color: rgba(#28334E, 0.5);
+  /* Background UI colors */
+  --main-background-color: #fff;
+  --header-background-color: #fff;
+  --body-background-color: #fff;
+  --dropdown-background-color: #F0F1FF;
+
+  /* System UI colors */
+  --system-green: #54B489;
+  --system-red: #BD3039;
+  --system-yellow: #E0A300;
+
+  /* Trading UI colors */
+  --bids: #54B489;
+  --asks: #E85E59;
+
+  /* Text, Icons, Divider colors */
+  --primary-text-color: #171C29;
+  --text-contrast-color: #FFFFFF;
+  --secondary-text-color: #28334E;
+  --icons: #28334E;
+  --icons-contrast-color: #28334E;
+  --divider-color: rgba(#000, 0.1);
   --text-muted: rgba(#28334E, 0.5);
-  --table-header-color: var(--bright);
+  --input-border: rgba(#28334E, 0.3);
+
+  /* Call to action (CTA) color opacity */ 
+  --cta-color-opacity-48: rgba(#3C78E0, 0.48);
+
+  /* Call to action (CTA) contrast opacity */  
+  --cta-contrast-opacity-48: rgba(#FFF, 0.48);
+  --cta-contrast-opacity-0-1: rgba(#3C78E0, 0.1);
```

Comment:
name of this value is totally wrong

--cta-contrast-opacity-12 is correct one

Please, check all values in this file and do by system -12, -24, -36, so on